docs-site/app/global.css

```
@@ -277,3 +476,1774 @@
+  display: grid;
+  grid-template-columns: 3.5rem minmax(max-content, 1fr) 1.5rem;
+  min-height: 1.75em;
+  padding-right: 1rem;
+}
+
+.scenario-line-reveal {
+  position: absolute;
+  inset: 0;
+  z-index: -1;
+  transform-origin: left;
+  background: color-mix(in oklab, currentColor 4%, transparent);
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)